keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -283,6 +286,9 @@ func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]stri
 			},
 		},
 	}
+	if runtime.cgroupDriver == v1alpha2.CGroupDriverSystemd {
+		psc.Linux.CgroupParent = "/pods.slice/pods-copyresource.slice/pods-copyresource-pod.slice"
```

Comment:
```suggestion
		psc.Linux.CgroupParent = "/kubeedge-setup.slice/pods-copyresource.slice"
```

Reply:
Finally decided to use "/kubeedge.slice/kubeedge-setup.slice/kubeedge-setup-podcopyresource.slice"